src/twisted/protocols/amp.py

```diff
@@ -291,6 +291,9 @@ def trapZero(result):
 ]
 
 
+_T_Callable = TypeVar("_T_Callable", bound=Callable[..., object])
```

Comment:
some places use `_Tc` we should be consistent

Reply:
Just asking.
Can we have these defined inside something like `twisted.python.typing` and import the types from there ?

I hope this can help with getting consistent type helpers.

Reply:
Yes you can do that, but it's best practice to define type variables each time you use them, this will be enforced with https://peps.python.org/pep-0695/

Reply:
Thanks for the info. https://peps.python.org/pep-0695/ looks good :)

we can live with the current code